### PR TITLE
add plugin: exec-all

### PR DESCRIPTION
**Pull request by jpdasma**
More details with regards to the plugin here: https://github.com/jpdasma/kubectl-exec-all

Sample usage:

```kubectl plugin exec-all daemonset docker -- docker system prune -a -f```

-----

**Checklist for plugin developers:**

- [/] Read the [Plugin Naming Guide](https://github.com/GoogleContainerTools/krew/tree/master/docs/NAMING_GUIDE.md)
- [/] Verify the installation from URL works (`kubectl plugin install --source`)

### CODE REVIEW

**Comment by jpdasma** [comment]
I signed it!

File: plugins/exec-all.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - uri: "https://github.com/jpdasma/kubectl-exec-all/archive/v1.0.0.zip"
+    sha256: "93f4da0b9127833924ede21d1ac1452a189e23b54339d8eeac1623a89dbfbd17"
+    files:
+    - from: "/*/unix/*"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [linux]}
+  version: "v1.0.0"
+  shortDescription: "Execute a command in all of the container in a specified resource"
```
Comment (ahmetb) [inline plugins/exec-all.yaml]: typo: containers.

**Comment by ahmetb** [comment]
This looks fantastic! Found a small typo.

File: plugins/exec-all.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+spec:
+  platforms:
+  - uri: "https://github.com/jpdasma/kubectl-exec-all/archive/v1.0.0.zip"
+    sha256: "93f4da0b9127833924ede21d1ac1452a189e23b54339d8eeac1623a89dbfbd17"
+    files:
+    - from: "/*/unix/*"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [linux]}
```
Comment (ahmetb) [inline plugins/exec-all.yaml]: Would this not work on macOS as well? If so, please add `darwin` to this list.

File: plugins/exec-all.yaml (hunk 1 above)
Comment (jpdasma) [reply plugins/exec-all.yaml]: fixed, thanks!

File: plugins/exec-all.yaml (hunk 2 above)
Comment (jpdasma) [reply plugins/exec-all.yaml]: added darwin

**Comment by ahmetb** [comment]
Locally verified. Thanks.

**Comment by ahmetb** [comment]
@jpdasma  Do you mind signing the CNCF CLA at the issue https://github.com/kubernetes/org/issues/599 so we can complete the transfer of the index repository to Kubernetes as a sub-project?

**Comment by ahmetb** [comment]
@jpdasma Friendly reminder to see if you can sign the CNCF CLA.

**Pull request merged**